python/py-python-socks/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-python-socks
+version             2.0.3
+revision            0
+license             Apache2
```

Comment:
this should be `Apache-2` as `port lint` tells you

Reply:
Fixed thanks.